reporter.go

```diff
@@ -8,59 +8,49 @@ import (
 	"time"
 
 	"github.com/aws/aws-lambda-go/lambdacontext"
-	"github.com/rcrowley/go-metrics"
-	"github.com/wavefronthq/go-metrics-wavefront"
+	wavefront "github.com/wavefronthq/wavefront-sdk-go/senders"
 )
 
-// incrementCounter increments the counter by the given value if report is true
-func incrementCounter(counter metrics.Counter, value int64, report bool) {
+// updateMetric increments the value of a given metric if report is true
+func updateMetric(metric *float64, inc int64, report bool) {
```

Comment:
As suggested below, once below change is  made , we don't really need the check for reportStandardMetrics inside updateMetric(..).